dts/bindings/gpio/x-powers,axp192-gpio.yaml

```diff
@@ -18,6 +18,8 @@ compatible: "x-powers,axp192-gpio"
 
 include: gpio-controller.yaml
 
+on-bus: axp192
```

Comment:
can you please explain why this is a bus?

Reply:
I found this way in the max31790 implementation.
https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/bindings/mfd/maxim%2Cmax31790.yaml#L10

I adopted it because I think it is a smart way to describe the MFD device and each function's dependencies.
But if this is not a good idea, I'll revert it.